doc/languages-frameworks/r.section.md

```diff
@@ -94,6 +94,54 @@ Executing `nix-shell` will then drop you into an environment equivalent to the
 one above. If you need additional packages just add them to the list and
 re-enter the shell.
 
+## torch with GPU acceleration {#torch-gpu-accel}
+
+The `torch` package included in `nixpkgs` is based on the binary packages by the
+mlverse team. By default, the CPU version of the package is installed. On Linux
+distributions other than NixOS, it is possible to enable GPU acceleration by
+setting `config.cudaSupport` to true when importing `nixpkgs` in a `shell.nix`.
+You will also need to use `nixglhost`. Here is an example of such a shell:
+
+```nix
+{ pkgs ? import <nixpkgs> {config.cudaSupport = true; }, lib ? pkgs.lib }:
+
+let
+ nixglhost-sources = pkgs.fetchFromGitHub {
+   owner = "numtide";
+   repo = "nix-gl-host";
+   rev = "main";
+   # Replace this with the hash Nix will complain about, TOFU style.
+   hash = "";
+ };
```

Comment:
I think this should be documented in its own chapter rather than in the R section: https://github.com/NixOS/nixpkgs/pull/264433/ (has been stale for a while, didn't get the chance to follow up) but maybe it's ok to temporarily merge this section and then replace it with a short reference when the chapter is added